gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java

```diff
@@ -0,0 +1,40 @@
+package io.trino.gateway.ha.security;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.trino.gateway.ha.security.LbOAuthManager.OidcTokens;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+public class TestOidcToken {
+  @Test
+  public void testOidcTokenParams() {
+    OidcTokens oidcTokens = null;
+    try {
+      ObjectMapper objectMapper = new ObjectMapper();
+
+      String jsonStr = "{\"id_token\" : \"ABC235234\", "
+          + "\"access_token\" : \"AcessABCD123\", "
+          + "\"refresh_token\" : \"RefreshTKN\", "
+          + "\"token_type\" : \"TOKENType\", "
+          + "\"expires_in\" : \"123456\", "
+          + "\"to_be_ignored\" : \"XYX123456\", "
```

Comment:
Can we somehow make it more explicit what this test actually does .. e.g. it verifies that unknown parameters dont trip up parsing

Reply:
should be good with the last check in.